catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/monitor/impl/WorkspaceServiceImpl.java

```diff
@@ -120,6 +132,33 @@ public String toString() {
         .orElse(Collections.emptyList());
   }
 
+  @Override
+  public List<QueryMetacardImpl> getQueryMetacards(WorkspaceMetacardImpl workspaceMetacard) {
+    List<String> queryIds = workspaceMetacard.getQueries();
+
```

Comment:
❓ Should we bail out early if there were no queryIds?

Reply:
I have no personal preference. The query wouldn't find any results to bailing out early is fine with me. Also gives us a chance to log this behavior.